### PR TITLE
Chap1 assg

### PR DESCRIPTION
**Pull request by sritej-reddy**

### CODE REVIEW

File: c-unit1-assignments/src/Assignment_02.cpp [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+	// digits (first digit in array at position 0) then i am rightshifting by one place and appending that carry
+	// at positon 0;
+	factorialDigits[0] = 1;
+	if (n == 0 || n == 1)
+		return 1;
+	int i = 0, count = 0, temp = 0, carry = 0;
+	while (n>1)
+	{
+		count = i;
+		while (i >= 0)
```
Comment (krushiraj) [inline c-unit1-assignments/src/Assignment_02.cpp]: Oka while ni oka separate function ga rasko better.

**Review by krushiraj — CHANGES_REQUESTED** [review]
Just change that one code. Everything else is okay!

File: c-unit1-assignments/src/Assignment_02.cpp (hunk 1 above)
Comment (sritej-reddy) [reply c-unit1-assignments/src/Assignment_02.cpp]: Done. closes #18

Comment (sritej-reddy) [reply c-unit1-assignments/src/Assignment_02.cpp]: anna ayindi commit chesina

File: c-unit1-assignments/src/Assignment_02.cpp [hunk 2]
```diff
@@ -24,40 +24,49 @@ void rightShift(int a[],int n,int pos)
...
+		carry = temp / 10;
+		factorialDigits[i] = temp % 10;
+		if (i == 0 && carry)
+		{
+			count=extendArray(carry, count, factorialDigits);
+		}
+		i--;
+	}
+	return count;
+}
```
Comment (krushiraj) [inline c-unit1-assignments/src/Assignment_02.cpp:55]: This looks great now.

**Review by krushiraj — APPROVED** [review]
Merging this.

**Pull request merged**